editors/sced/README.md

```diff
@@ -48,7 +48,7 @@ Place wrapper-scripts for sclang and scsynth into `/usr/local/bin`.
 
 ```
 #!/bin/sh
-cd /full/path/to/SuperCollider.app/Contents/Resources
+cd /full/path/to/SuperCollider.app/Contents/MacOS
```

Comment:
Seems it went through various permutations of wrong and right, so that question is hard to anwer. Atm it looks right to me. Wrong?